app/assets/stylesheets/pghero/application.css

```diff
@@ -272,10 +272,18 @@ hr {
   background-color: #5bc0de;
 }
 
+.alert-info-citus {
```

Comment:
I'm hesitant to change color theme and project name without prior discussion, can you remove those from this PR. Otherwise it is fine.

Reply:
OK...